pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -67,6 +67,7 @@ const (
 	ImagePusherRoleName       = "system:image-pusher"
 	ImageBuilderRoleName      = "system:image-builder"
 	ImagePrunerRoleName       = "system:image-pruner"
+	ImageSignerRoleName       = "system:image-signer"
```

Comment:
> We want project admins to be able to manage image signatures. Since this is so related to registry should we also grant privs to registry-admin role?

You can't do that with a cluster-scoped resource, you'll need to project access at a project scope.